test/e2e/common/volumes.go

```diff
@@ -75,7 +75,7 @@ var _ = framework.KubeDescribe("GCP Volumes", func() {
 	////////////////////////////////////////////////////////////////////////
 
 	framework.KubeDescribe("NFSv4", func() {
-		It("should be mountable for NFSv4 [Volume]", func() {
+		It("should be mountable for NFSv4 [sig-storage]", func() {
```

Comment:
SIGDescribe is a package-level function in the storage/ directory, so it's not available here.